camelot/view/action_steps/gui.py

```diff
@@ -30,22 +30,23 @@
 """
 Various ``ActionStep`` subclasses that manipulate the GUI of the application.
 """
+import functools
+import json
 from typing import Any, List, Tuple
 
-from dataclasses import dataclass
-
-from ...core.qt import QtCore, QtWidgets, is_deleted
-
-
+from dataclasses import dataclass, field
 
 from camelot.admin.action.base import ActionStep
 from camelot.core.exception import CancelRequest
 from camelot.core.utils import ugettext_lazy as _
 from camelot.view.controls import editors
 from camelot.view.controls.standalone_wizard_page import StandaloneWizardPage
+from ...core.qt import QtCore, QtWidgets, is_deleted
+from ...core.serializable import DataclassSerializable
+
 
 @dataclass
-class UpdateEditor(ActionStep):
+class UpdateEditor(ActionStep, DataclassSerializable):
```

Comment:
@jeffh92 @Student2-vFinance : ik denk nt dat dit gaat werken in contexten waar de value itself niet serialiseerbaar is, haal in deze step aub de DataclassSerializable weg, zodat ik de andere wijzigingen reeds kan mergen .. thx